### ISSUE TITLE
Widget showing GitHub instead of widget

### ISSUE THREAD

**Issue by Preston2291** [LeonStraathof/pfsense-speedtest-widget#8]
Hello, 
I followed all steps and when I went to add the widget it appears to be linking the github from the php file instead of the widget. Attached is the screenshot. I downloaded the speedtest.widget.php from the github but maybe got the wrong thing? Any help would be nice. Thanks. 
<img width="469" alt="Screenshot 2024-11-04 at 6 28 14 PM" src="https://github.com/user-attachments/assets/b20ca89c-5186-4e0a-948b-14fc4daec18c">

**Comment by ruanmed** [comment]
Hi @Preston2291.

Seems like instead of saving the raw file `speedtest.widget.php`, you [navigated to the file using GitHub file browser](https://github.com/LeonStraathof/pfsense-speedtest-widget/blob/main/speedtest.widget.php) and saved the HTML page of GitHub itself.

You must save the raw file and upload that instead. You can click on the "Download raw file" button to ensure you are saving the proper file before uploading.

![image](https://github.com/user-attachments/assets/dca527d2-c52e-4f07-b19d-55f1e6f12938)

**Comment by Danie10** [comment]
Yes, I did the same. After downloading the raw file and uploading that, then just doing to mv command, it all came right. This issue can be closed.

**Comment by ThisIsStupidGitHub** [comment]
I also did the same. Thanks
